### DISCUSSION TITLE
Please add your favorite book from the 20th century to the list in the README and make Pull Request . (reminder)

### DISCUSSION THREAD

**Discussion by KiyotadaMori** [NAIST-SE/AttractingContributors#1192] [2024 Attraction Contributions]
Please add your favorite book from the 20th century to the list in the README and make Pull Request .

The Strict Rule

Share only 20th-century books

Link

https://github.com/KiyotadaMori/AttractingContributors-20th_century_books

How to

git remote add 20-books https://github.com/KiyotadaMori/AttractingContributors-20th_century_books
git fetch 20-books
git checkout -b add-books-YOURNAME 20-books/main
Add your favourite books.
git add README.md
git commit -m "add 20-books"
git push origin add-books-YOURNAME
click "Compare and pull request" and "Create Pull Reques

**Comment by marshr00m** [comment]
I submitted a pull request. I would appreciate any additions to my list!
https://github.com/marshr00m/AttractingContributors-Where-would-you-like-to-travel
